android/app/src/main/kotlin/com/pockeat/MainActivity.kt

```
@@ -13,110 +15,137 @@ import com.pockeat.widget.CustomHomeWidgetPlugin
+    private fun handleLaunchUri(call: MethodCall, result: Result) {
+        try {
+            val uriString = call.argument<String>("uri")
+            if (uriString != null) {
+                val intent = Intent(Intent.ACTION_VIEW, Uri.parse(uriString))
+                intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+                startActivity(intent)
+                result.success(true)
+            } else {
+                result.error("NULL_URI", "URI cannot be null", null)
+            }
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)